sql/queries.sql

```diff
@@ -110,12 +110,27 @@ CREATE TABLE authors ( name text, id bigint );
 WITH long_names AS ( SELECT id FROM authors WHERE char_length(name) > 15 )
 SELECT title FROM articles;
 
--- joins are unsupported
-SELECT title, authors.name FROM authors, articles WHERE authors.id = articles.author_id;
+--  queries which involve functions in FROM clause are unsupported.
+select * from  articles, position('om' in 'Thomas');
```

Comment:
Use uppercase for SQL keywords: `SELECT`, `FROM`, etc. As a general rule, mimic the style of whatever file you're editing.